tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -125,6 +134,19 @@ def test_poly1d_order_leading_zeros(self, xp, dtype):
         a = xp.array([0, 0, 1, 2, 3, 0], dtype)
         return xp.poly1d(a).order
 
+    @testing.for_signed_dtypes()
+    @testing.numpy_cupy_allclose(rtol=1e-6)
+    @_wraps_root_test
+    def test_poly1d_roots1(self, xp, dtype):
+        a = xp.array([-3, -2.5, 3], dtype)
+        return xp.poly1d(a).roots
```

Comment:
Why not sort the return value directly for the readability?
```suggestion
    @testing.for_signed_dtypes()
    @testing.numpy_cupy_allclose(rtol=1e-6)
    def test_poly1d_roots1(self, xp, dtype):
        a = xp.array([-3, -2.5, 3], dtype)
        out = xp.poly1d(a).roots
        # The current `cupy.roots` doesn't guarantee the order of results.
        return xp.sort(out)
```